pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -434,6 +438,13 @@ func (m *cgroupManagerImpl) toResources(resourceConfig *ResourceConfig) *libcont
 			Limit:    uint64(0),
 		})
 	}
+	// only for cgroup v2
```

Comment:
one thing that may become confusing is when `Unified` should be used as opposed to existing `Resources` config from runc libcontainer. As I understand, some of the existing properties like `Memory` (https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/opencontainers/runc/libcontainer/configs/cgroup_linux.go#L49) already map to existing cgroupv2 items like `memory.Max`. 

However for the rest of the cgroupv2 options that are not implicitly converted underneath in `Resources` struct, `Unified` should be used instead. Is my understanding here correct?

/cc @giuseppe

Reply:
Yes, that's right.

Reply:
ideally unified is used for all the resources when running on cgroup v2.  It doesn't make difference for the memory.max limit, but for e.g. for the cpu controller you can specify the correct setting without relying on the conversions performed by the OCI runtime